clients/drcachesim/tests/cache_replacement_policy_unit_test.cpp

```diff
@@ -136,6 +136,54 @@ unit_test_cache_lru_four_way()
     cache_lru_test.access_and_check_cache(ADDRESS_E, 2); // A E c D
 }
 
+void
+unit_test_cache_lru_eight_way()
+{
+    cache_policy_test_t<cache_lru_t> cache_lru_test(/*associativity=*/8,
+                                                    /*line_size=*/64,
+                                                    /*total_size=*/1024);
+    cache_lru_test.initialize_cache();
+
+    const addr_t ADDRESS_A = 0;
+    const addr_t ADDRESS_B = 128;
+    const addr_t ADDRESS_C = 256;
+    const addr_t ADDRESS_D = 384;
+    const addr_t ADDRESS_E = 512;
+    const addr_t ADDRESS_F = 640;
+    const addr_t ADDRESS_G = 768;
+    const addr_t ADDRESS_H = 896;
+    const addr_t ADDRESS_I = 1024;
+    const addr_t ADDRESS_J = 1152;
+    const addr_t ADDRESS_K = 1280;
+    const addr_t ADDRESS_L = 1408;
```

Comment:
This looks like the exact same set as in the 8-way FIFO test.  Please share these rather than duplicating.  Maybe a single shared vector (in cache_policy_test_t?), with the names A through L as indices (like my suggestion in the prior PR).

Reply:
Done.